js/client_exporter/image_creator.js

```diff
@@ -541,7 +539,7 @@ function drawCanvasElements(elements, context, parentOptions) {
 function setLineDash(context, options) {
     var matches = options["stroke-dasharray"] && options["stroke-dasharray"].match(/(\d+)/g);
 
-    if(matches && matches.length && context.setLineDash) { // IE10 does not have setLineDash
```

Comment:
@roman-simionov 
>&& context.setLineDash

Are we sure that this part is specific for IE10 only?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/setLineDash

Reply:
@roman-simionov @alexander-kotov-dx 
What about FF for android?

Reply:
android 8, FF 60 - it works